debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java

```diff
@@ -532,7 +532,9 @@ public MySqlConnectionConfiguration(Configuration config) {
                     .without("queryInterceptors");
 
             this.jdbcConfig = JdbcConfiguration.adapt(jdbcConfigBuilder.build());
-            String driverClassName = this.jdbcConfig.getString(MySqlConnectorConfig.JDBC_DRIVER);
+            // The jdbcConfig has unprefixed values so you have to access it without the prefix.
```

Comment:
@gurpiarbassi You should be able to obtain the driver class from `config` variable instead of hard-coding the `jdbc.driver` string.

Reply:
@jpechane thanks, I've corrected this.